pages/_app.tsx

```diff
@@ -1,10 +1,32 @@
-import { AppProps } from 'next/app'
 import '../styles/globals.scss'
-
+// import 'bootstrap/dist/css/bootstrap.min.css'
+import Layout from '../components/Layout'
+import Head from 'next/head'
+import { AppProps } from 'next/app'
 
 function MyApp({ Component, pageProps }: AppProps) {
-
-  return <Component {...pageProps} />
+  return (
+    <Layout>
+      <Head>
+        <title>Upward</title>
+        <meta name="description" content="Advancing women in tech" />
+        <link rel="icon" href="/upward-logo.png" />
+        <script
+          src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"
+          integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p"
+          crossOrigin="anonymous"></script>
+        <script
+          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.min.js"
+          integrity="sha384-Atwg2Pkwv9vp0ygtn1JAojH0nYbwNJLPhwyoVbhoPwBhjQPR5VtM2+xf0Uwh9KtT"
+          crossOrigin="anonymous"></script>
+        {/* <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.min.js" integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s" crossorigin="anonymous"></script>
```

Comment:
commented out code. Should we keep it?

Reply:
I want to keep this. This is for bootstrap4... incase someone wants to use things from 4 instead of 5.